solr/core/src/java/org/apache/solr/packagemanager/PackageManager.java

```diff
@@ -228,20 +230,20 @@ public List<SolrPackageInstance> fetchInstalledPackageInstances() throws SolrExc
   public Map<String, SolrPackageInstance> getPackagesDeployed(String collection) {
     Map<String, String> packages = null;
     try {
-      NavigableObject result =
-          (NavigableObject)
-              Utils.executeGET(
-                  solrClient.getHttpClient(),
-                  solrBaseUrl
-                      + PackageUtils.getCollectionParamsPath(collection)
-                      + "/PKG_VERSIONS?omitHeader=true&wt=javabin",
-                  Utils.JAVABINCONSUMER);
+      Map<String, String[]> paramsMap = new LinkedHashMap<>();
+      paramsMap.put("omitHeader", new String[] {"true"});
+      paramsMap.put("wt", new String[] {"javabin"});
+      NamedList<Object> result =
+          solrClient.request(
+              new GenericSolrRequest(
+                  SolrRequest.METHOD.GET,
+                  PackageUtils.getCollectionParamsPath(collection) + "/PKG_VERSIONS",
+                  new ModifiableSolrParams(paramsMap)));
       packages =
           (Map<String, String>)
               result._get("/response/params/PKG_VERSIONS", Collections.emptyMap());
-    } catch (PathNotFoundException ex) {
-      // Don't worry if PKG_VERSION wasn't found. It just means this collection was never touched by
-      // the package manager.
```

Comment:
Do we need to keep this comment?   Did we lose the fact that it's okay to not have a PKG_VERSIONS?